### PR TITLE
[Stationboard] Change tooltip for simple point

### PR DESCRIPTION
**Pull request by pfanguin**

### CODE REVIEW

**Comment by pfanguin** [comment]
Test link : https://mf-geoadmin3.dev.bgdi.ch/update_oev_tooltip/
@mariokeusen and @davidoesch : can you test and give me a feedback ? Thanks

**Comment by davidoesch** [comment]
Tooltip dies not work

**Comment by pfanguin** [comment]
It's work for me

Tooltip simple point : 

https://mf-geoadmin3.dev.bgdi.ch/update_oev_tooltip/?ch.bav.haltestellen-oev=8515561&lang=fr&topic=ech&showTooltip=true&bgLayer=ch.swisstopo.pixelkarte-farbe&layers=ch.bav.haltestellen-oev,ch.swisstopo.zeitreihen,ch.bfs.gebaeude_wohnungs_register,ch.bafu.wrz-wildruhezonen_portal,ch.swisstopo.swisstlm3d-wanderwege&layers_visibility=true,false,false,false,false&layers_timestamp=,18641231,,,&X=152971.00&Y=536253.00&zoom=9

Tootlip standard: 

https://mf-geoadmin3.dev.bgdi.ch/update_oev_tooltip/?ch.bav.haltestellen-oev=8592105&lang=fr&topic=ech&showTooltip=true&bgLayer=ch.swisstopo.pixelkarte-farbe&layers=ch.bav.haltestellen-oev,ch.swisstopo.zeitreihen,ch.bfs.gebaeude_wohnungs_register,ch.bafu.wrz-wildruhezonen_portal,ch.swisstopo.swisstlm3d-wanderwege&layers_visibility=true,false,false,false,false&layers_timestamp=,18641231,,,&X=152877.00&Y=536122.00&zoom=9

File: chsdi/templates/htmlpopup/oev_haltestellen.mako [hunk 1]
```diff
@@ -129,6 +143,7 @@ $(document).ready(function() {
         <td id="departures${id}" class="col-departures"></td>
         <td id="timeDiff${id}" class="col-time-diff"></td>
     </tr>
+% endif :
```
Comment (loicgasser) [inline chsdi/templates/htmlpopup/oev_haltestellen.mako]: oups typo please remove the `:`

File: buildout_branch.cfg.in [hunk 2]
```diff
@@ -9,7 +9,7 @@ apache_base_path = ${git_branch}
 api_url = //mf-chsdi3.${vars:deploy_target}.bgdi.ch/${git_branch}
 host = mf-chsdi3.${vars:deploy_target}.bgdi.ch
 # On the branch, adapt this to point to the correct mf-geoadmin instance
-geoadminhost = mf-geoadmin3.${vars:deploy_target}.bgdi.ch/${git_branch}
+geoadminhost = mf-geoadmin3.${vars:deploy_target}.bgdi.ch/update_oev_tooltip
```
Comment (loicgasser) [inline buildout_branch.cfg.in]: please revert, this is only supposed to be a template

**Comment by loicgasser** [comment]
It works for me as well.

Just remove this commit https://github.com/geoadmin/mf-chsdi3/commit/69d222926235f88469dae4164bef9f051ce07fb3

and fix the typo and we're good to go.

**Comment by pfanguin** [comment]
Thanks Loïc, it's done

**Comment by davidoesch** [comment]
Works for me content is OK (retina display issue seems to be solved)

**Pull request merged**